pom.xml

```diff
@@ -68,7 +68,17 @@
         <dependency>
             <groupId>edu.illinois.cs</groupId>
             <artifactId>testrunner-maven-plugin</artifactId>
-            <version>1.0</version>
+            <!-- TODO: update dependency -->
+            <version>1.2-SNAPSHOT</version>
```

Comment:
Since this depends so heavily on updates to testrunner, let's finish the changes in testrunner first and release those changes before having to update here. I would prefer not to depend on SNAPSHOT versions, especially when it's code from other repositories.

Reply:
Sure, let's first update the testrunner plugin first.

Reply:
Yes, the version that iDFlakies depends on should be 1.1 (testrunner-maven-plugin is currently set to 1.1-SNAPSHOT, so we will release version 1.1 there once @luoos's changes there are merged)

Reply:
Updated the version